spec/datadog/core/telemetry/http/transport_spec.rb

```diff
@@ -27,9 +27,10 @@
     let(:env) { instance_double(Datadog::Core::Telemetry::Http::Env, body: payload, path: path) }
     let(:headers) do
       {
-        Datadog::Core::Telemetry::Http::Ext::HEADER_CONTENT_TYPE => 'application/json',
-        Datadog::Core::Telemetry::Http::Ext::HEADER_DD_TELEMETRY_API_VERSION => 'v1',
-        Datadog::Core::Telemetry::Http::Ext::HEADER_DD_TELEMETRY_REQUEST_TYPE => request_type,
+        'Content-Type' => 'application/json',
+        'DD-Telemetry-API-Version' => 'v1',
+        'DD-Telemetry-Request-Type' => 'app-started',
+        'DD-Internal-Untraced-Request' => '1',
```

Comment:
Curious why you decided to remove the constants. Not opposed to the change, I just want to know your reasoning 😄

Reply:
When you use to the System Under Test (SUT) as test assertions, you can end up with test assertions that are couple to your SUT.

This means you could have wrong assertions, because they are simply reflecting something incorrect in your SUT, and regurgitating it back to you in a test case.

This doesn't mean we should replace every reference from the SUT from our test assertions, but when feasible, it's good to avoid it.

In a few cases, it can create unmaintainable tests, and it might be fine to use the SUT, but I tend to avoid it if possible.